docs/zh/sdk-manual/wechat.md

```diff
@@ -0,0 +1,317 @@
+| _session_start_timestamp |  是   | 会话起始时间                                                                             |
+| _session_duration        |  是   | 会话持续时间                                                                             |
+| _session_number          |  是   | 会话自增数，从1开始                                                                         |
+| _screen_name             |  否  | 当前页面名称                                                                             |
+| _screen_id               |  否  | 当前页面ID                                                                             |
+| _screen_route            |  否  | 当前页面URL                                                                            |
+| _previous_screen_name    |  否  | 上一页面名称                                                                             |
+| _previous_screen_id      |  否  | 上一页面ID                                                                             |
+| _previous_screen_route   |  否  | 上一页面URL                                                                            |
+| _engagement_time_msec    |  否  | 用户在当前页面停留时间                                                                        |
+| _is_first_time           |  否  | 用户首次使用微信小程序时为`true` |
+
+### Item属性
+
+| 属性名称          | 数据类型             | 是否必需 | 说明        |
+|---------------|------------------|------|-----------|
+| id            | string           | 否    | item的id   |
+| name          | string           | 否   | item的名称   |
+| brand         | string           | 否   | item的品牌   |
+| price         | string \| number | 否   | item的价格   |
+| quantity      | number           | 否   | item的数量   |
+| creative_name | string           | 否   | item的创意名称 |
+| creative_slot | string           | 否   | item的创意槽  |
+| location_id   | string           | 否   | item的位置id |
+| category      | string           | 否   | item的类别   |
+| category2     | string           | 否   | item的类别2  |
+| category3     | string           | 否   | item的类别3  |
+| category4     | string           | 否   | item的类别4  |
+| category5     | string           | 否   | item的类别5  |
+
+### 事件示例
+
+```json
+{
+  "app_id": "appId",
+  "unique_id": "c84ad28d-16a8-4af4-a331-f34cdc7a7a18",
+  "device_id": "be4b3f1e-b2a8-4b7b-9055-4257e3e313c8",
+  "event_type": "_screen_view",
+  "event_id": "460daa08-0717-4385-8f2e-acb5bd019ee7",
+  "timestamp": 1667877566697,
+  "platform": "WeChatMP",
+  "os_name": "iOS",
+  "os_version": "iOS 12.0.1",
+  "wechat_version": "8.0.5",
+  "wechat_sdk_version": "3.0.0",
+  "brand": "devtools",
+  "model": "iPhone 6/7/8",
+  "system_language": "en",
+  "screen_height": 667,
+  "screen_width": 375,
+  "zone_offset": 28800000,
+  "network_type": "wifi",
+  "sdk_version": "0.2.0",
+  "sdk_name": "aws-solution-clickstream-sdk",
+  "app_version": "1.0.1",
+  "app_package_name": "wxbd614036ba3d1f05",
+  "user": {
```

Comment:
Let's add an item example to show the structure of the items array

Reply:
done